pixeltable/utils/console_output.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+def get_level(level_str: str, default: int = logging.INFO) -> int:
+    """
+    Simple function to get logging level from string
+
+    Args:
+        level_str: Level string (e.g., 'INFO', 'DEBUG')
+        default: Default level to use if conversion fails
+
+    Returns:
+        Logging level as integer
+    """
+    try:
+        return getattr(logging, level_str.upper())
```

Comment:
perhaps only support debug, info, warn anything above warn is always going to console?